intel_extension_for_transformers/llm/runtime/graph/__init__.py

```diff
@@ -126,7 +127,7 @@ def init(self, model_name, use_quant=True, use_gptq=False, **quant_kwargs):
         # clean
         os.remove(fp32_bin)
 
-    def init_from_bin(self, model_type, model_path, **generate_kwargs):
```

Comment:
I think double underline stands for enforced private. In this way, this change breaks the API (and will break the ppl script AFAIK) and it need further discuses.

Reply:
recover it